server/gokbg3/grails-app/controllers/org/gokb/rest/PackageController.groovy

```diff
@@ -665,6 +667,51 @@ class PackageController {
     render result as JSON
   }
 
+  @Secured(['IS_AUTHENTICATED_ANONYMOUSLY'])
+  def jobs() {
```

Comment:
Similar to the methods in `CuratoryGroupController` for example, the parameters `max` and `offset` have to be handled.

Reply:
handeled